AppServices/Profiles/ProductProfile.cs

```diff
@@ -8,6 +8,8 @@ public class ProductProfile : Profile
     {
         public ProductProfile()
         {
+            CreateMap<Product, ProductResult>();
+            CreateMap<ProductResult, Product>();
```

Comment:
Em que cenário isso ocorre? `CreateMap<ProductResult, Product>();`
```suggestion
            CreateMap<Product, ProductResult>().ReverseMap();
```

Reply:
Não ocorre. Já retirei